includes/widgets/video.php

```diff
@@ -271,6 +271,18 @@ protected function register_controls() {
 			]
 		);
 
+		$this->add_control(
+			'youtube_lazy_load',
```

Comment:
Change the control name to `lazy_load` to allow future implementations in other providers